### DISCUSSION TITLE
BOTOCORE_ACCEPTED_KWARGS - Add arguments to put_object

### DISCUSSION THREAD

**Discussion by Laony88** [aws/aws-sdk-pandas#812] [Q&A]
Hello, I am Laon P. 

awswrangler library limits arguments for put_object within following list when using 'awswrangler.s3.to_csv' function. 
(Is it right?)

```BOTOCORE_ACCEPTED_KWARGS: Dict[str, Set[str]] = {
   "put_object": {
        "ACL",
        "Metadata",
        "ServerSideEncryption",
        "StorageClass",
        "SSECustomerAlgorithm",
        "SSECustomerKey",
        "SSEKMSKeyId",
        "SSEKMSEncryptionContext",
        "Tagging",
        "RequestPayer",
        "ExpectedBucketOwner",
    },
```

But, I need to add 'ContentEncoding' argument as a system defined argument that boto3 supports.
So, I want to know why this library limits the arguments, and it is possible to add additional argument that is not listed but is supported from boto3.
https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3.html#S3.Client.put_object

**Comment by jaidisido** [comment]
Solved with this [PR](https://github.com/awslabs/aws-data-wrangler/pull/832), will be available in next release
